src/main/java/biz/netcentric/maven/extension/repofromenv/FromEnvReposConfigurationProcessor.java

```diff
@@ -209,7 +220,7 @@ private Server getServer(RepoFromEnv repoFromEnv) {
     void addImplicitFileRepo(List<RepoFromEnv> reposFromEnv, File multiModuleProjectDirectory) {
         File implicitRepo = new File(multiModuleProjectDirectory, IMPLICIT_FILE_REPO_PATH);
         if(implicitRepo.exists()) {
-            reposFromEnv.add(new RepoFromEnv(IMPLICIT_FILE_REPO_ID, implicitRepo.toURI().toString(), null, null));
+            reposFromEnv.add(0, new RepoFromEnv(IMPLICIT_FILE_REPO_ID, implicitRepo.toURI().toString(), null, null));
```

Comment:
this ensures that if a file repo is configured it is ordered before remote repos (as a file repo will always be the most performant source)